core/src/main/res/drawable/ic_history_24px.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ Kiwix Android
+  ~ Copyright (c) 2020 Kiwix <android.kiwix.org>
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program. If not, see <http://www.gnu.org/licenses/>.
+  ~
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Shouldn't we have night mode icons?

Reply:
They are automatically inverted, I don't know why? 
![Screenshot_1595920887](https://user-images.githubusercontent.com/22193232/88632327-ba354c80-d0b3-11ea-91a1-a06971476882.png)

Reply:
well that is surprising... wahey, less work